src/main/java/com/searchservice/app/rest/ManageTableResource.java

```diff
@@ -182,10 +181,10 @@ public ResponseEntity<Response> undoTable(@RequestParam int tenantId, @PathVaria
 		} else {
 
 			log.debug("Exception Occured While Performing Restore Delete For Table: {} ", tableNameForMessage);
-			throw new BadRequestOccurredException(400, tableNameForMessage + " is not available for restoring");
+			throw new BadRequestOccurredException(400,"Something Went Wrong While Performing Restore for "+TABLE+ tableNameForMessage);
```

Comment:
Take the StatusCode & Message from HttpStatusCode here, if appropriate message & code not present contact to Mentor/TL to _add those in_ **_HttpStatusCode_** class